### PR TITLE
Update Storybook to v6.4

### PR DESCRIPTION
**Pull request by jandrade**
## Summary:

This will enable us to use Storybook in the near future with:

- Interactive tests
- Faster tooling (on-demand architecture)
- ESLint support for writing stories using best practices/recommendations.
- Feature flags (?) why not!

Issue: XXX-XXXX

## Test plan:

- Run `yarn install -W && yarn start:storybook"

- Verify that the SB environment continues working without problems.

### CODE REVIEW

**Review by jeresig — APPROVED** [review]
Yay! Excited for the potential in this new release!

File: packages/wonder-blocks-clickable/src/components/clickable.stories.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default {
     title: "Navigation/Clickable",
 };
 
-export const basic: StoryComponentType = () => (
+export const Basic: StoryComponentType = () => (
```
Comment (jandrade) [inline packages/wonder-blocks-clickable/src/components/clickable.stories.js:18]: This naming is now enforced by the linter.

File: .eslintrc.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ module.exports = {
         "@khanacademy",
         // This config includes rules from @testing-library/jest-dom as well
         "plugin:testing-library/react",
+        // This config includes rules from storybook to enforce story best
+        // practices
+        "plugin:storybook/recommended",
```
Comment (kevinbarabash) [inline .eslintrc.js:9]: nice!

**Review by kevinbarabash — APPROVED** [review]
LGTM.  I'm excited to see all the new features in action.  Thanks for upgrading. 🚀

**Pull request merged**